lib/internal/test_runner/test.js

```diff
@@ -209,6 +210,19 @@ class TestContext {
     return subtest.start();
   }
 
+  retries(number) {
+    validateNumber(number, 'number', 0, NumberMAX_SAFE_INTEGER);
```

Comment:
is `Infinity` supported?

Reply:
I don't think `Infinity` should be supported, as then the test will *never* fail (which undermines the point of a test). Despite MAX being, well, giant, it is finite.